src/components/ErrorMessage.tsx

```diff
@@ -0,0 +1,40 @@
+import React, { useEffect } from 'react';
+import cn from 'classnames';
+
+type Props = {
+  errorMessage: string;
+  setErrorMessage: (errorMessage: string) => void;
+};
+
+export const Error: React.FC<Props> = ({ errorMessage, setErrorMessage }) => {
+  useEffect(() => {
+    if (errorMessage) {
+      const timeOut = setTimeout(() => {
+        setErrorMessage('');
+      }, 3000);
+
+      return () => {
+        clearTimeout(timeOut);
+      };
+    } else {
+      return;
+    }
+  }, [errorMessage, setErrorMessage]);
+
+  return (
+    <div
+      data-cy="ErrorNotification"
+      className={cn('notification is-danger is-ligth has-text-weight-normal', {
+        hidden: !errorMessage,
+      })}
+    >
+      <button
+        data-cy="HideErrorButton"
+        type="button"
+        className="delete"
+        onClick={() => setErrorMessage('')}
```

Comment:
Consider creating handler function here

Reply:
Not fixed 
<img width="874" alt="image" src="https://github.com/user-attachments/assets/4fb32816-c9b6-46e5-ba57-627e6ffefde6">